config/daemon/systemd.md

```diff
@@ -94,6 +99,16 @@ you need to add this configuration in the Docker systemd service file.
     [Service]
     Environment="HTTPS_PROXY=https://proxy.example.com:443/"
     ```
+     
+    Note: for [rootless](https://docs.docker.com/engine/security/rootless/) docker, 
+    create following files, with similar contents as described above:
+    - `~/.config/systemd/user/docker.service.d/http-proxy.conf`
+      
+      
+    Or,
+      
+      
+    - `~/.config/systemd/user/docker.service.d/https-proxy.conf`
```

Comment:
Not something new introduced in this PR, but I think we should consider showing that multiple environment variables can be set in a single file. There's no need to create two separate files for this.

Reply:
yes we can merge the two files, however the file locations are different for rootless and rootful (regular) mode